docs/platform/howto/manage-org-vpc-peering-azure.md

```diff
@@ -409,10 +441,10 @@ from your Azure VNet to the Aiven organization VPC:
 1. Check if the status of the peering connection is `ACTIVE`:
 
    ```bash
-   avn vpc peering-connection get -v             \
-     --project-vpc-id $aiven_organization_vpc_id \
-     --peer-cloud-account $user_subscription_id  \
-     --peer-resource-group $user_resource_group  \
+   avn vpc peering-connection get -v                  \
```

Comment:
This command does not exist in the CLI PR, `organization vpc get` also shows states and properties of peering connections.